save-core/src/jvmTest/kotlin/org/cqfn/save/core/utils/ProcessBuilderInternalTest.kt

```diff
@@ -0,0 +1,102 @@
+package org.cqfn.save.core.utils
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+@Suppress("LOCAL_VARIABLE_EARLY_DECLARATION",
+    "SAY_NO_TO_VAR",
+    "MISSING_KDOC_TOP_LEVEL",
+    "MISSING_KDOC_CLASS_ELEMENTS",
+    "MISSING_KDOC_ON_FUNCTION"
+)
+class ProcessBuilderInternalTest {
+    private val processBuilder = ProcessBuilder()
+
+    @Test
+    fun `check stdout`() {
+        val actualResult = processBuilder.exec("echo something", null)
+        val expectedCode = 0
+        val expectedStdout = "something"
+        val expectedStderr: List<String> = emptyList()
+        assertEquals(expectedCode, actualResult.code)
+        // posix `system()` and JVM process builder returns lines with different whitespaces, so we cut them
+        assertEquals(expectedStdout, actualResult.stdout[0].trimEnd())
+        assertEquals(expectedStderr, actualResult.stderr)
+    }
+
+    @Test
+    fun `check stdout with redirection`() {
+        val actualResult = processBuilder.exec("echo something >/dev/null", null)
+        var expectedCode: Int
+        val expectedStdout = ""
+        lateinit var expectedStderr: List<String>
+        when {
+            System.getProperty("os.name").contains("Linux", ignoreCase = true) ||
```

Comment:
`!isCurrentOsWindows()`?

Reply:
I think it could be done in https://github.com/cqfn/save/issues/53. 
For now we can only check whether current OS == Windows, or not.

However, the behavior of MacOS is different, than in Linux, at least in process exit codes.
Thus, after support of MacOS we will need some other form of this function, which will give us information not only, that current OS is Windows, but concrete answer, which OS is used.

Reply:
ok, but in this particular case you have the same branch for Linux and MacOS, so I suggested to unify them.

Reply:
Changed